### PR TITLE
US-516218 : Constellation UI helm chart updates.

### PR DESCRIPTION
**Pull request by anuds20**
Updated the constellationUI helm charts. Updated the readme according to the current on-prem customer deployment model.

### CODE REVIEW

**Comment by anuds20** [comment]
@npjohnson637 Can you please review this PR?

File: charts/backingservices/charts/constellation/README.md [hunk 1]
```diff
@@ -1,26 +1,73 @@
-# Constellationui helm chart
+# Constellation UI
```
Comment (npjohnson637) [inline charts/backingservices/charts/constellation/README.md:1]: @anuds20 , this description here must start with a link to our primary doc: https://documents.constellation.pega.io/static/88/introduction.html . It should not be duplicating content from that doc to here.

Only have material that is specific to this install here.

**Review by npjohnson637 — CHANGES_REQUESTED** [review]
Hi Santosh,

We should not duplicate doc that is already in our main install guide.
This description here must start with a link to our primary doc: https://documents.constellation.pega.io/static/88/introduction.html . It should not be duplicating content from our main doc to here.

Only have material that is specific to this install here.

File: charts/backingservices/charts/constellation/values.yaml [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
     registry:
-      url: "cirrus-docker.jfrog.io"
-      username: cameng2
-      password: d8538894e2
+      url: YOUR_DOCKER_REGISTRY_URL
+      username: YOUR_DOCKER_REGISTRY_USERNAME
+      password: YOUR_DOCKER_REGISTRY_PASSWORD
     # Docker image information for the Pega docker image, containing the application server.
     constellation:
       image: cirrus-docker.jfrog.io/constellation-appstatic-service/docker-image:1.0.8-20221228123724
```
Comment (taz-pega-work) [inline charts/backingservices/charts/constellation/values.yaml]: @anuds20 @npjohnson637 - is it wise to hard code the Docker image AND TAG in this readme? What if it changes? I think the right thing to do is reference the Docker image name in the readme in a manner that follows the rest of the service team's Docker image references in the other yaml charts>>
pega values.yaml>>
pega:
      image: "pegasystems/pega"
# Elasticsearch deployment settings.
pegasearch:
  image: "pegasystems/search"
installer:
  image: "YOUR_INSTALLER_IMAGE:TAG"
hazelcast:
  image: "YOUR_HAZELCAST_IMAGE:TAG"
  clusteringServiceImage: "YOUR_CLUSTERING_SERVICE_IMAGE:TAG"

**Review by taz-pega-work — COMMENTED** [review]
Please review the specific tagging of the docker image image. 

I think you should add a reference in the main constellation readme to the Downloading Docker images docs so clients know how to get the best image for Constellation>>https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html

File: charts/backingservices/charts/constellation/values.yaml (hunk 2 above)
Comment (anuds20) [reply charts/backingservices/charts/constellation/values.yaml]: @taz-pega-work Made the changes according to your comments. Also added a link to downloading pega docker images as requested.

**Review by npjohnson637 — APPROVED** [review]
Nice, thanks.

Nigel

File: charts/backingservices/charts/constellation/README.md [hunk 3]
```diff
@@ -1,43 +1,49 @@
...
+To download the constellation appstatic service image, specify the version and use the following command
+
+```bash
+$ sudo docker pull pega-docker.downloads.pega.com/constellation-appstatic-service/docker-image:xxxxxxx
+
+Digest: <encryption verification>
+Status: Downloaded pega-docker.downloads.pega.com/constellation-appstatic-service/docker-image:xxxxxxx
+```
+
+For details about downloading and then pushing Docker images to your repository for your deployment, see [Using Pega-provided Docker images](https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images).
```
Comment (taz-pega-work) [inline charts/backingservices/charts/constellation/README.md:33]: this is the correct link to the doc now>>
[Pega-provided Docker images](https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html)

**Review by taz-pega-work — CHANGES_REQUESTED** [review]
only one change left...refer to the latest docs.pega.com content. everything else is great. thanks for your collaboration.

**Comment by sudeshjethoe** [comment]
can this fix be integrated? it seems the constellation deployment is broken currently

File: charts/backingservices/charts/constellation/README.md (hunk 3 above)
Comment (anuds20) [reply charts/backingservices/charts/constellation/README.md:33]: @taz-pega-work Can you please review and approve?

**Review by kingakowalska1 — APPROVED** [review]
Looks good, thanks.

**Review by taz-pega-work — APPROVED** [review]
thanks for making the change here, @santosh!! Ready to go. Let's ship it!! ;-)

**Pull request merged**